tests/test_bgp_af.rb

```diff
@@ -610,6 +610,102 @@ def test_dampening
   # rubocop:enable Metrics/AbcSize,Metrics/MethodLength
 
   ##
+  ## maximum_paths
+  ##
+  def maximum_paths(asn, vrf, af)
+    /ipv4/.match(af) ? af = %w(ipv4 unicast) : af = %w(ipv6 unicast)
+    bgp_af = RouterBgpAF.new(asn, vrf, af)
+
+    #
+    # Set and verify
+    #
+    val = 7
+    bgp_af.maximum_paths = val
+    assert_equal(bgp_af.maximum_paths, val,
+                 'Error: maximum paths value not match to set value')
+
+    val = 9
+    bgp_af.maximum_paths = val
+    assert_equal(bgp_af.maximum_paths, val,
+                 'Error: maximum paths value not match to set value')
+
+    val = bgp_af.default_maximum_paths
+    bgp_af.maximum_paths = val
+    assert_equal(bgp_af.maximum_paths, val,
+                 'Error: maximum paths value not match to default value')
+  end
+
+  def test_maximum_paths
```

Comment:
I know you got this code pattern from other places in this file but in this case I'd prefer that you simplify it. You can replace everything in this method with something like this:

```
vrfs = ['default', 'red']
afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
vrfs.each do |vrf|
  afs.each do |af|
    maximum_paths(55, vrf, af)
  end
end
```